dev/bots/run_command.dart

```diff
@@ -182,7 +192,12 @@ Future<CommandResult> runCommand(String executable, List<String> arguments, {
 
   final CommandResult result = await command.onExit;
 
-  if ((result.exitCode == 0) == expectNonZeroExit || (expectedExitCode != null && result.exitCode != expectedExitCode)) {
+  final bool skipErrorExit = expectNoTests &&
```

Comment:
Is this because of an auto failure when no tests are run by the the underlying command? It seems weird that a generic 'runCommand' function knows anything about tests.

Reply:
Yes. Currently a test run that runs no tests fails by default, and there's no way to know ahead of time (without trying to run them) that there are no tests to run.  I created an issue, but I don't know when/if they'll address it, so I added this here as a workaround. https://github.com/dart-lang/test/issues/1546

Reply:
Got it. Perhaps put a link to this request here so we can remove if they fix the issue for us?

Reply:
Done.